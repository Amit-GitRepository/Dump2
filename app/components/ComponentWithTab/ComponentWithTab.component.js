import ComponentWithTabPage from '../../pages/ComponentWithTab/ComponentWithTab.page';
import PropTypes from 'prop-types';
import React, {Component} from 'react';

class ComponentWithTab extends Component {
  renderWithTab () {}
  render () {
    return (
      <ComponentWithTabPage scrollEnabled={this.props.scrollEnabled}>
        {this.renderWithTab()}
      </ComponentWithTabPage>
    );
  }
}

ComponentWithTab.defaultProps = {
  scrollEnabled: true
};

ComponentWithTab.propTypes = {
  scrollEnabled: PropTypes.bool
};
export default ComponentWithTab;
