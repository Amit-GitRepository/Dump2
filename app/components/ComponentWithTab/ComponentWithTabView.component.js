import NavFooter from '../NavFooter/NavFooter.component';
import PropTypes from 'prop-types';
import React, {Component} from 'react';
import styles from './ComponentWithTabView.style';
import {ScrollView, View} from 'react-native';

class ComponentWithTabView extends Component {
  render () {
    const {goToScreen, currentScreen, isLoggedIn, children, scrollEnabled} = this.props;
    const contentContainerStyles = [styles.contentContainer, isLoggedIn ? styles.loggedInPadding : null];
    return (
      <View style={styles.container}>
        <ScrollView scrollEnabled={scrollEnabled}
          showsVerticalScrollIndicator={false}
          showsHorizontalScrollIndicator={false}
          keyboardShouldPersistTaps='handled'
          contentContainerStyle={contentContainerStyles}
        >
          {children}
        </ScrollView>
        {isLoggedIn ? <NavFooter goToScreen={goToScreen} currentScreen={currentScreen}/> : null}
      </View>
    );
  }
}

ComponentWithTabView.defaultProps = {
  scrollEnabled: true
};

ComponentWithTabView.propTypes = {
  currentScreen: PropTypes.string,
  goToScreen: PropTypes.func,
  isLoggedIn: PropTypes.bool,
  children: PropTypes.node,
  scrollEnabled: PropTypes.bool
};
export default ComponentWithTabView;
