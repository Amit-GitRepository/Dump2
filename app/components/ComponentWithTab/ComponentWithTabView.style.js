export default {
  container: {
    flex: 1
  },
  contentContainer: {
    flexGrow: 1
  },
  loggedInPadding: {
    paddingBottom: 65
  }
};
