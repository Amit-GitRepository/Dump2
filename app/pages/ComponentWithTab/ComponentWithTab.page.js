import ComponentWithTabView from '../../components/ComponentWithTab/ComponentWithTabView.component';
import getCurrentRouteName from 'redux-ga-screen-tracker/utils/transformer.utils';
import isEmpty from 'lodash/isEmpty';
import PropTypes from 'prop-types';
import React, {Component} from 'react';
import result from 'lodash/result';
import {connect} from 'react-redux';
import {NavigationActions} from 'react-navigation';

class ComponentWithTabPage extends Component {
  render () {
    const {goToScreen, profile, routes, children} = this.props;
    const isLoggedIn = !isEmpty(profile);
    const currentScreen = getCurrentRouteName(routes[0]);
    return (
      <ComponentWithTabView goToScreen={goToScreen} currentScreen={currentScreen} isLoggedIn={isLoggedIn}>
        {children}
      </ComponentWithTabView>
    );
  }
}

ComponentWithTabPage.propTypes = {
  profile: PropTypes.object,
  goToScreen: PropTypes.func,
  routes: PropTypes.array,
  children: PropTypes.node
};

export const mapStateToProps = ({user, nav}) => ({
  profile: result(user, 'profile'),
  routes: result(nav, 'routes')
});

export const mapDispatchToProps = (dispatch) => ({
  goToScreen: (screen) => {
    dispatch(NavigationActions.navigate({routeName: screen}));
  }
});
export default connect(mapStateToProps, mapDispatchToProps)(ComponentWithTabPage);
