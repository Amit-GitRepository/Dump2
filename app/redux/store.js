import createSagaMiddleware from 'redux-saga';
import middlewares from './middlewares/index.middleware';
import reduxPromise from 'redux-promise';
import rootReducer from './reducers/index.reducer';
import sagas from './sagas/index.saga';
import screenTracking from 'redux-ga-screen-tracker';
import {applyMiddleware, compose, createStore} from 'redux';
import {getScreenTrackingConfig} from '../utils/analytics.util';

const composeEnhancers = global.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ ? global.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__({}) : compose;

const sagaMiddleware = createSagaMiddleware();
const enhancer = composeEnhancers(applyMiddleware(sagaMiddleware, reduxPromise, screenTracking(getScreenTrackingConfig()), ...middlewares));

export default function getStore (initalState = {}) {
  const store = createStore(rootReducer, initalState, enhancer);
  if (module.hot) {
    // BREAKING change in : https://github.com/reactjs/react-redux/releases/tag/v2.0.0
    module.hot.accept(() => {
      store.replaceReducer(rootReducer);
    });
  }
  sagaMiddleware.run(sagas);
  return store;
}
