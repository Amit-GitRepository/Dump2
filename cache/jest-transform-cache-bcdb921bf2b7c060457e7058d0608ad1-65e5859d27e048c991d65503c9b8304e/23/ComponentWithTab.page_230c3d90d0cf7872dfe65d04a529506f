Object.defineProperty(exports,"__esModule",{value:true});exports.mapDispatchToProps=exports.mapStateToProps=undefined;var cov_tvw7dozym=function(){var path='/Users/amitkumar/iTrueService/DevCode/iservice-app/app/pages/ComponentWithTab/ComponentWithTab.page.js',hash='8c93e067de675dafbc24a207101aadcfb6909ee6',global=new Function('return this')(),gcv='__coverage__',coverageData={path:'/Users/amitkumar/iTrueService/DevCode/iservice-app/app/pages/ComponentWithTab/ComponentWithTab.page.js',statementMap:{'0':{start:{line:12,column:52},end:{line:12,column:62}},'1':{start:{line:13,column:23},end:{line:13,column:40}},'2':{start:{line:14,column:26},end:{line:14,column:56}},'3':{start:{line:15,column:4},end:{line:19,column:6}},'4':{start:{line:23,column:0},end:{line:28,column:2}},'5':{start:{line:30,column:31},end:{line:33,column:2}},'6':{start:{line:30,column:49},end:{line:33,column:1}},'7':{start:{line:35,column:34},end:{line:39,column:2}},'8':{start:{line:35,column:49},end:{line:39,column:1}},'9':{start:{line:37,column:4},end:{line:37,column:62}}},fnMap:{'0':{name:'(anonymous_0)',decl:{start:{line:11,column:2},end:{line:11,column:3}},loc:{start:{line:11,column:12},end:{line:20,column:3}},line:11},'1':{name:'(anonymous_1)',decl:{start:{line:30,column:31},end:{line:30,column:32}},loc:{start:{line:30,column:49},end:{line:33,column:1}},line:30},'2':{name:'(anonymous_2)',decl:{start:{line:35,column:34},end:{line:35,column:35}},loc:{start:{line:35,column:49},end:{line:39,column:1}},line:35},'3':{name:'(anonymous_3)',decl:{start:{line:36,column:14},end:{line:36,column:15}},loc:{start:{line:36,column:26},end:{line:38,column:3}},line:36}},branchMap:{},s:{'0':0,'1':0,'2':0,'3':0,'4':0,'5':0,'6':0,'7':0,'8':0,'9':0},f:{'0':0,'1':0,'2':0,'3':0},b:{},_coverageSchema:'332fd63041d2c1bcb487cc26dd0d5f7d97098a6c'},coverage=global[gcv]||(global[gcv]={});if(coverage[path]&&coverage[path].hash===hash){return coverage[path];}coverageData.hash=hash;return coverage[path]=coverageData;}();var _createClass=function(){function defineProperties(target,props){for(var i=0;i<props.length;i++){var descriptor=props[i];descriptor.enumerable=descriptor.enumerable||false;descriptor.configurable=true;if("value"in descriptor)descriptor.writable=true;Object.defineProperty(target,descriptor.key,descriptor);}}return function(Constructor,protoProps,staticProps){if(protoProps)defineProperties(Constructor.prototype,protoProps);if(staticProps)defineProperties(Constructor,staticProps);return Constructor;};}();var _ComponentWithTabView=require('../../components/ComponentWithTab/ComponentWithTabView.component');var _ComponentWithTabView2=_interopRequireDefault(_ComponentWithTabView);
var _transformer=require('redux-ga-screen-tracker/utils/transformer.utils');var _transformer2=_interopRequireDefault(_transformer);
var _isEmpty=require('lodash/isEmpty');var _isEmpty2=_interopRequireDefault(_isEmpty);
var _propTypes=require('prop-types');var _propTypes2=_interopRequireDefault(_propTypes);
var _react=require('react');var _react2=_interopRequireDefault(_react);
var _result=require('lodash/result');var _result2=_interopRequireDefault(_result);
var _reactRedux=require('react-redux');
var _reactNavigation=require('react-navigation');function _interopRequireDefault(obj){return obj&&obj.__esModule?obj:{default:obj};}function _classCallCheck(instance,Constructor){if(!(instance instanceof Constructor)){throw new TypeError("Cannot call a class as a function");}}function _possibleConstructorReturn(self,call){if(!self){throw new ReferenceError("this hasn't been initialised - super() hasn't been called");}return call&&(typeof call==="object"||typeof call==="function")?call:self;}function _inherits(subClass,superClass){if(typeof superClass!=="function"&&superClass!==null){throw new TypeError("Super expression must either be null or a function, not "+typeof superClass);}subClass.prototype=Object.create(superClass&&superClass.prototype,{constructor:{value:subClass,enumerable:false,writable:true,configurable:true}});if(superClass)Object.setPrototypeOf?Object.setPrototypeOf(subClass,superClass):subClass.__proto__=superClass;}var

ComponentWithTabPage=function(_Component){_inherits(ComponentWithTabPage,_Component);function ComponentWithTabPage(){_classCallCheck(this,ComponentWithTabPage);return _possibleConstructorReturn(this,(ComponentWithTabPage.__proto__||Object.getPrototypeOf(ComponentWithTabPage)).apply(this,arguments));}_createClass(ComponentWithTabPage,[{key:'render',value:function render()
{cov_tvw7dozym.f[0]++;var _ref=(cov_tvw7dozym.s[0]++,
this.props),goToScreen=_ref.goToScreen,profile=_ref.profile,routes=_ref.routes,children=_ref.children;
var isLoggedIn=(cov_tvw7dozym.s[1]++,!(0,_isEmpty2.default)(profile));
var currentScreen=(cov_tvw7dozym.s[2]++,(0,_transformer2.default)(routes[0]));cov_tvw7dozym.s[3]++;
return(
_react2.default.createElement(_ComponentWithTabView2.default,{goToScreen:goToScreen,currentScreen:currentScreen,isLoggedIn:isLoggedIn},
children));


}}]);return ComponentWithTabPage;}(_react.Component);cov_tvw7dozym.s[4]++;


ComponentWithTabPage.propTypes={
profile:_propTypes2.default.object,
goToScreen:_propTypes2.default.func,
routes:_propTypes2.default.array,
children:_propTypes2.default.node};cov_tvw7dozym.s[5]++;


var mapStateToProps=exports.mapStateToProps=function mapStateToProps(_ref2){var user=_ref2.user,nav=_ref2.nav;cov_tvw7dozym.f[1]++;cov_tvw7dozym.s[6]++;return{
profile:(0,_result2.default)(user,'profile'),
routes:(0,_result2.default)(nav,'routes')};
};cov_tvw7dozym.s[7]++;

var mapDispatchToProps=exports.mapDispatchToProps=function mapDispatchToProps(dispatch){cov_tvw7dozym.f[2]++;cov_tvw7dozym.s[8]++;return{
goToScreen:function goToScreen(screen){cov_tvw7dozym.f[3]++;cov_tvw7dozym.s[9]++;
dispatch(_reactNavigation.NavigationActions.navigate({routeName:screen}));
}};
};exports.default=
(0,_reactRedux.connect)(mapStateToProps,mapDispatchToProps)(ComponentWithTabPage);