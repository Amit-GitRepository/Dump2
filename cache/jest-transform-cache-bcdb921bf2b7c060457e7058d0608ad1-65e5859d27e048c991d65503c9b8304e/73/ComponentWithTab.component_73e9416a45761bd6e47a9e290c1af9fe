Object.defineProperty(exports,"__esModule",{value:true});var cov_2hkgmmb7r6=function(){var path='/Users/amitkumar/iTrueService/DevCode/iservice-app/app/components/ComponentWithTab/ComponentWithTab.component.js',hash='683e47e674843f66c29c1ee74963296b475fbe5b',global=new Function('return this')(),gcv='__coverage__',coverageData={path:'/Users/amitkumar/iTrueService/DevCode/iservice-app/app/components/ComponentWithTab/ComponentWithTab.component.js',statementMap:{'0':{start:{line:7,column:4},end:{line:11,column:6}}},fnMap:{'0':{name:'(anonymous_0)',decl:{start:{line:5,column:2},end:{line:5,column:3}},loc:{start:{line:5,column:19},end:{line:5,column:21}},line:5},'1':{name:'(anonymous_1)',decl:{start:{line:6,column:2},end:{line:6,column:3}},loc:{start:{line:6,column:12},end:{line:12,column:3}},line:6}},branchMap:{},s:{'0':0},f:{'0':0,'1':0},b:{},_coverageSchema:'332fd63041d2c1bcb487cc26dd0d5f7d97098a6c'},coverage=global[gcv]||(global[gcv]={});if(coverage[path]&&coverage[path].hash===hash){return coverage[path];}coverageData.hash=hash;return coverage[path]=coverageData;}();var _createClass=function(){function defineProperties(target,props){for(var i=0;i<props.length;i++){var descriptor=props[i];descriptor.enumerable=descriptor.enumerable||false;descriptor.configurable=true;if("value"in descriptor)descriptor.writable=true;Object.defineProperty(target,descriptor.key,descriptor);}}return function(Constructor,protoProps,staticProps){if(protoProps)defineProperties(Constructor.prototype,protoProps);if(staticProps)defineProperties(Constructor,staticProps);return Constructor;};}();var _ComponentWithTab=require('../../pages/ComponentWithTab/ComponentWithTab.page');var _ComponentWithTab2=_interopRequireDefault(_ComponentWithTab);
var _react=require('react');var _react2=_interopRequireDefault(_react);function _interopRequireDefault(obj){return obj&&obj.__esModule?obj:{default:obj};}function _classCallCheck(instance,Constructor){if(!(instance instanceof Constructor)){throw new TypeError("Cannot call a class as a function");}}function _possibleConstructorReturn(self,call){if(!self){throw new ReferenceError("this hasn't been initialised - super() hasn't been called");}return call&&(typeof call==="object"||typeof call==="function")?call:self;}function _inherits(subClass,superClass){if(typeof superClass!=="function"&&superClass!==null){throw new TypeError("Super expression must either be null or a function, not "+typeof superClass);}subClass.prototype=Object.create(superClass&&superClass.prototype,{constructor:{value:subClass,enumerable:false,writable:true,configurable:true}});if(superClass)Object.setPrototypeOf?Object.setPrototypeOf(subClass,superClass):subClass.__proto__=superClass;}var

ComponentWithTab=function(_Component){_inherits(ComponentWithTab,_Component);function ComponentWithTab(){_classCallCheck(this,ComponentWithTab);return _possibleConstructorReturn(this,(ComponentWithTab.__proto__||Object.getPrototypeOf(ComponentWithTab)).apply(this,arguments));}_createClass(ComponentWithTab,[{key:'renderWithTab',value:function renderWithTab()
{cov_2hkgmmb7r6.f[0]++;}},{key:'render',value:function render()
{cov_2hkgmmb7r6.f[1]++;cov_2hkgmmb7r6.s[0]++;
return(
_react2.default.createElement(_ComponentWithTab2.default,null,
this.renderWithTab()));


}}]);return ComponentWithTab;}(_react.Component);exports.default=


ComponentWithTab;