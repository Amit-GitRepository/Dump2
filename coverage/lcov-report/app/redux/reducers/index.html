<!doctype html>
<html lang="en">
<head>
    <title>Code coverage report for app/redux/reducers</title>
    <meta charset="utf-8" />
    <link rel="stylesheet" href="../../../prettify.css" />
    <link rel="stylesheet" href="../../../base.css" />
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <style type='text/css'>
        .coverage-summary .sorter {
            background-image: url(../../../sort-arrow-sprite.png);
        }
    </style>
</head>
<body>
<div class='wrapper'>
  <div class='pad1'>
    <h1>
      <a href="../../../index.html">All files</a> app/redux/reducers
    </h1>
    <div class='clearfix'>
      <div class='fl pad1y space-right2'>
        <span class="strong">72.62% </span>
        <span class="quiet">Statements</span>
        <span class='fraction'>61/84</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">61.54% </span>
        <span class="quiet">Branches</span>
        <span class='fraction'>40/65</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">78.57% </span>
        <span class="quiet">Functions</span>
        <span class='fraction'>11/14</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">73.49% </span>
        <span class="quiet">Lines</span>
        <span class='fraction'>61/83</span>
      </div>
    </div>
  </div>
  <div class='status-line medium'></div>
<div class="pad1">
<table class="coverage-summary">
<thead>
<tr>
   <th data-col="file" data-fmt="html" data-html="true" class="file">File</th>
   <th data-col="pic" data-type="number" data-fmt="html" data-html="true" class="pic"></th>
   <th data-col="statements" data-type="number" data-fmt="pct" class="pct">Statements</th>
   <th data-col="statements_raw" data-type="number" data-fmt="html" class="abs"></th>
   <th data-col="branches" data-type="number" data-fmt="pct" class="pct">Branches</th>
   <th data-col="branches_raw" data-type="number" data-fmt="html" class="abs"></th>
   <th data-col="functions" data-type="number" data-fmt="pct" class="pct">Functions</th>
   <th data-col="functions_raw" data-type="number" data-fmt="html" class="abs"></th>
   <th data-col="lines" data-type="number" data-fmt="pct" class="pct">Lines</th>
   <th data-col="lines_raw" data-type="number" data-fmt="html" class="abs"></th>
</tr>
</thead>
<tbody><tr>
	<td class="file medium" data-value="billUsage.reducer.js"><a href="billUsage.reducer.js.html">billUsage.reducer.js</a></td>
	<td data-value="52.78" class="pic medium"><div class="chart"><div class="cover-fill" style="width: 52%;"></div><div class="cover-empty" style="width:48%;"></div></div></td>
	<td data-value="52.78" class="pct medium">52.78%</td>
	<td data-value="36" class="abs medium">19/36</td>
	<td data-value="30.43" class="pct low">30.43%</td>
	<td data-value="23" class="abs low">7/23</td>
	<td data-value="40" class="pct low">40%</td>
	<td data-value="5" class="abs low">2/5</td>
	<td data-value="54.29" class="pct medium">54.29%</td>
	<td data-value="35" class="abs medium">19/35</td>
	</tr>

<tr>
	<td class="file high" data-value="forceUpdate.reducer.js"><a href="forceUpdate.reducer.js.html">forceUpdate.reducer.js</a></td>
	<td data-value="100" class="pic high"><div class="chart"><div class="cover-fill cover-full" style="width: 100%;"></div><div class="cover-empty" style="width:0%;"></div></div></td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="5" class="abs high">5/5</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="4" class="abs high">4/4</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="1" class="abs high">1/1</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="5" class="abs high">5/5</td>
	</tr>

<tr>
	<td class="file high" data-value="msisdn.reducer.js"><a href="msisdn.reducer.js.html">msisdn.reducer.js</a></td>
	<td data-value="100" class="pic high"><div class="chart"><div class="cover-fill cover-full" style="width: 100%;"></div><div class="cover-empty" style="width:0%;"></div></div></td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="4" class="abs high">4/4</td>
	<td data-value="75" class="pct medium">75%</td>
	<td data-value="4" class="abs medium">3/4</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="1" class="abs high">1/1</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="4" class="abs high">4/4</td>
	</tr>

<tr>
	<td class="file high" data-value="networkStatus.reducer.js"><a href="networkStatus.reducer.js.html">networkStatus.reducer.js</a></td>
	<td data-value="100" class="pic high"><div class="chart"><div class="cover-fill cover-full" style="width: 100%;"></div><div class="cover-empty" style="width:0%;"></div></div></td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="4" class="abs high">4/4</td>
	<td data-value="66.67" class="pct medium">66.67%</td>
	<td data-value="3" class="abs medium">2/3</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="1" class="abs high">1/1</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="4" class="abs high">4/4</td>
	</tr>

<tr>
	<td class="file high" data-value="sideMenu.reducer.js"><a href="sideMenu.reducer.js.html">sideMenu.reducer.js</a></td>
	<td data-value="100" class="pic high"><div class="chart"><div class="cover-fill cover-full" style="width: 100%;"></div><div class="cover-empty" style="width:0%;"></div></div></td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="4" class="abs high">4/4</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="3" class="abs high">3/3</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="1" class="abs high">1/1</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="4" class="abs high">4/4</td>
	</tr>

<tr>
	<td class="file high" data-value="spinner.reducer.js"><a href="spinner.reducer.js.html">spinner.reducer.js</a></td>
	<td data-value="100" class="pic high"><div class="chart"><div class="cover-fill cover-full" style="width: 100%;"></div><div class="cover-empty" style="width:0%;"></div></div></td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="6" class="abs high">6/6</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="6" class="abs high">6/6</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="1" class="abs high">1/1</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="6" class="abs high">6/6</td>
	</tr>

<tr>
	<td class="file medium" data-value="storeLocator.reducer.js"><a href="storeLocator.reducer.js.html">storeLocator.reducer.js</a></td>
	<td data-value="68.75" class="pic medium"><div class="chart"><div class="cover-fill" style="width: 68%;"></div><div class="cover-empty" style="width:32%;"></div></div></td>
	<td data-value="68.75" class="pct medium">68.75%</td>
	<td data-value="16" class="abs medium">11/16</td>
	<td data-value="54.55" class="pct medium">54.55%</td>
	<td data-value="11" class="abs medium">6/11</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="3" class="abs high">3/3</td>
	<td data-value="68.75" class="pct medium">68.75%</td>
	<td data-value="16" class="abs medium">11/16</td>
	</tr>

<tr>
	<td class="file high" data-value="user.reducer.js"><a href="user.reducer.js.html">user.reducer.js</a></td>
	<td data-value="88.89" class="pic high"><div class="chart"><div class="cover-fill" style="width: 88%;"></div><div class="cover-empty" style="width:12%;"></div></div></td>
	<td data-value="88.89" class="pct high">88.89%</td>
	<td data-value="9" class="abs high">8/9</td>
	<td data-value="81.82" class="pct high">81.82%</td>
	<td data-value="11" class="abs high">9/11</td>
	<td data-value="100" class="pct high">100%</td>
	<td data-value="1" class="abs high">1/1</td>
	<td data-value="88.89" class="pct high">88.89%</td>
	<td data-value="9" class="abs high">8/9</td>
	</tr>

</tbody>
</table>
</div><div class='push'></div><!-- for sticky footer -->
</div><!-- /wrapper -->
<div class='footer quiet pad2 space-top1 center small'>
  Code coverage
  generated by <a href="https://istanbul.js.org/" target="_blank">istanbul</a> at Tue Oct 31 2017 11:24:42 GMT+0530 (IST)
</div>
</div>
<script src="../../../prettify.js"></script>
<script>
window.onload = function () {
        if (typeof prettyPrint === 'function') {
            prettyPrint();
        }
};
</script>
<script src="../../../sorter.js"></script>
</body>
</html>
