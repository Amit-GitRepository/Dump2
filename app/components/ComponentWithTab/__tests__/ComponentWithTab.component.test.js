import ComponentWithTab from '../ComponentWithTab.component';
import React from 'react';
import ReactSixteenAdapter from 'enzyme/build/adapters/ReactSixteenAdapter';
import renderer from 'react-test-renderer';
import {configure, shallow} from 'enzyme';
import {createStore} from 'redux';
import {Provider} from 'react-redux';

configure({adapter: new ReactSixteenAdapter()});
const store = createStore(() => ({
  user: {
    profile: {}
  },
  nav: {
    routes: []
  }
}));

describe('ComponentWithTab component', () => {
  it('renders correctly', () => {
    const wrapper = shallow(
      <Provider store={store}>
        <ComponentWithTab/>
      </Provider>
    );
    const component = renderer.create(<Provider store={store}>
      <ComponentWithTab/>
    </Provider>).toJSON();
    expect(component).toMatchSnapshot();
    expect(wrapper).toBeDefined();
  });
});
