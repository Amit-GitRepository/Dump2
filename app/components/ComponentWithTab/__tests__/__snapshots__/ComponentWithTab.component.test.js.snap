// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ComponentWithTab component renders correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
    }
  }
>
  <RCTScrollView
    contentContainerStyle={
      Array [
        Object {
          "flexGrow": 1,
        },
        null,
      ]
    }
    keyboardShouldPersistTaps="handled"
    scrollEnabled={true}
    showsHorizontalScrollIndicator={false}
    showsVerticalScrollIndicator={false}
  >
    <View />
  </RCTScrollView>
</View>
`;
